### ISSUE TITLE
Error when adding feed

### ISSUE THREAD

**Issue by pintman** [rss2email/rss2email#43]
I get the following error when following the README.

```
$ r2e new email@example.com
$ r2e add eff https://www.eff.org/rss/updates.xml
Traceback (most recent call last):
  File "/home/XXX/tmp/rss2email-ve/bin/r2e", line 5, in <module>
    rss2email.main.run()
  File "/home/XXX/tmp/rss2email-ve/lib/python3.5/site-packages/rss2email/main.py", line 172, in run
    args.func(feeds=feeds, args=args)
  File "/home/XXX/tmp/rss2email-ve/lib/python3.5/site-packages/rss2email/command.py", line 51, in add
    feed = feeds.new_feed(name=args.name, url=args.url, to=args.email, from_email=args.from_email)
AttributeError: 'Namespace' object has no attribute 'from_email'

```

**Comment by Profpatsch** [comment]
Hi pintman, I’m sorry, I merged a few commits I shouldn’t have merged. If you try master again now, it should work again.

**Comment by kaashif** [comment]
This issue was definitely fixed when the changes from #42 were reverted back in February. Or at least it works now.
